usaspending_api/awards/models_matviews/base_mv_model.py

```diff
@@ -5,7 +5,15 @@
 from usaspending_api.awards.models import Award
 
 
-class UniversalAwardView(models.Model):
+class BaseAwardSearchModel(models.Model):
```

Comment:
I feel like this file should be named `base_award_search.py`
Reasons
- generally I don't think we have a convention of suffixing with `_model` (file) or `Model` (class), because the location of the module and class _within_ a `models` folder/package gives that indication
- I think you could take out `mv` for matview, because this particular class doesn't really ever become a matview. It's "baseness" is about it providing the model structure and fields to other models. How those other models manifest and are persisted in the db is more their issue
- should add `award_search` indicating what business purpose this model is providing a base definition of

Reply:
done